libcalico-go/lib/backend/model/block_affinity.go

```diff
@@ -100,6 +100,10 @@ func (options BlockAffinityListOptions) KeyFromDefaultPath(path string) Key {
 	}
 	cidrStr := strings.Replace(r[0][2], "-", "/", 1)
 	_, cidr, _ := net.ParseCIDR(cidrStr)
+	if cidr == nil {
+		log.Debugf("Failed to parse CIDR in block affinity path: %q", path)
```

Comment:
Fuzzer found a latent bug

Reply:
I think you could hit this by manually creating a bad entry in etcd (in an etcd system).  It is UTed by the fuzz entry (all the listed Fuzz seed entries are run as unit tests).

Reply:
So just to check that I'm understanding: the fuzzer found a bug and told you the input needed to hit that, and then you added that input as one of the entries in `interestingPaths` ?

Reply:
Yes, exactly.  If we were running fuzz tests nightly, we'd get them to save their own corpus to somewhere persistent.  Then, any "previously failed" entries int he corpus are run first to make the tests deterministically fail.  Since we're not doing that, copying the bad entry back into the seed corpus seemed like a good stop gap.